modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,310 @@
 
 [[options]]
 == Options
 
-...
+=== Changed
+
+[[highlight-on-focus]]
+==== `highlight_on_focus`
+
+Previously, the default value for the editor configuration option `highlight_on_focus` was set to `false` by default. In {productname} 7.0 this option is now by default set to `true`
+
+As a result, the focus outline for the editor will by default be displayed when the focus is on the editor.
+
+Customers who have created their own custom skins, or have implemented their own outline and wish to not to use the default one will need to add the `highlight_on_focus: false` option to their {productname} configuration.
+
+[NOTE]
+This change does not impact editors using inline mode.
+
+=== Removed
+
+[[force-hex-color]]
+==== `force_hex_color`
+
+Previously in {productname} 6, all colors in the content HTML where set to use `RGB` values by default. As the common practice is using `hex` values this change will be **reverted**.
+
+.Example
+[source, html]
+----
+<!-- example using RGB values -->
+<p>Hello <span style="color: rgb(255, 0, 0);">red</span> color</p>
+
+<!-- example using HEX values -->
+<p>Hello <span style="color: #ff0000">red</span> color</p>
+----
+
+In {productname} 7.0, the default will be reverted back to using `hex` values by default.
+
+[IMPORTANT]
+The option `force_hex_color` will be removed altogether in {productname} 7.0.
+
+[[deprecated-options]]
+== Deprecated Options
+
+=== Changed
+
+[[deprecated-options-remove-trailing-brs-option-removed-from-our-DomParser-API]]
+==== `remove_trailing_brs` option removed from our `DomParser` API
+
+The `remove_trailing_brs` option was moved to the serializer and the existing option is no longer the best way to remove the relevant tags.
+
+[TIP]
+While the capability still exists for users that have added this option to their {productname} configuration, it is recommended that customers using our `DomParser` API directly, should migrate to use our `DomSerializer` setting instead.
 
 [[plugins]]
 == Plugins
 
-...
+=== Changed
+
+[[The-list-plugin-is-now-required-to-be-added-to-TinyMCE-config-in-order-to-execute-list-commands]]
+==== The list plugin is now required to be added to {productname} config, in order to execute list commands
+
+A breaking change has been introduced regarding the insertion of **lists** using the `InsertOrderedList` and `InsertUnorderedList` commands.
+
+Despite the presence of these commands, they have been functional even without the `list` plugin being added to the {productname} init configuration.
+
+{productname} 7.0, now **requires** that the `list` plugin is present in the `init` configuration for executing these insert list commands.
+
+As a result, and users that notice styling issues with their existing documents will be required to add the `list` plugin to their `init` config, as executing the `InsertOrderedList` and `InsertUnorderedList` commands will have longer effect the editor content.
```

Comment:
```suggestion
As a result, any users that notice styling issues with their existing documents will be required to add the `list` plugin to their `init` config, as executing the `InsertOrderedList` and `InsertUnorderedList` commands will have longer effect the editor content.
```

Reply:
This was updated in a previous commit, and the context has now been changed. "marking as resolved to un-block merging."